source/presentation/4.0/model.md

```diff
@@ -509,89 +509,94 @@ Properties...
 }
 ```
 
-
-
-
-
-
-
-
-
-
-
 #### Lights
-
-
-
+ 
 ##### Ambient Light
 
 `"type": "AmbientLight"`
 
-Ambient Lights evenly illuminates all objects in the scene, and does not have a direction or position.
-
+Ambient Light evenly illuminates all objects in the Scene, and does not have a direction or position.
```

Comment:
Are these "An Ambient Light", "A Point Light", etc? Not sure which reads better.

Reply:
I followed other examples in the document.  I'm sure we'll need to review for style later.